src/rmind/callbacks/loggers.py

```diff
@@ -44,6 +55,11 @@ def _call(self, trainer: pl.Trainer, pl_module: pl.LightningModule) -> None:
         ):
             return
 
+        if (self._every_n_batch is not None) and (
+            trainer.global_step % self._every_n_batch != 0
```

Comment:
batch-based hooks take a `batch_idx` -- maybe use that instead of `trainer.global_step`?